kornia/geometry/conversions.py

```diff
@@ -321,8 +321,21 @@ def rotation_matrix_to_angle_axis(rotation_matrix: torch.Tensor) -> torch.Tensor
 
     if not rotation_matrix.shape[-2:] == (3, 3):
         raise ValueError(f"Input size must be a (*, 3, 3) tensor. Got {rotation_matrix.shape}")
```

Comment:
Probably we should update this docs and checking len == 3 as -er what says above

Reply:
:+1: